torch/csrc/jit/serialization/unpickler.cpp

```diff
@@ -316,21 +317,21 @@ PickleOpCode Unpickler::readInstruction() {
       stack_.emplace_back(int64_t(value));
     } break;
     case PickleOpCode::BININT2: {
-      uint16_t value = read<uint16_t>();
+      uint16_t value = le16toh(read<uint16_t>());
       stack_.emplace_back(int64_t(value));
     } break;
     case PickleOpCode::BININT: {
-      int32_t value = read<int32_t>();
+      int32_t value = le32toh(read<int32_t>());
```

Comment:
Hmm, wouldn't it raise the warning, as `le32toh` expects unsigned input and returns unsigned type as result

Reply:
No warnings here for me. It's converted implicitly.